docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  app:
```

Comment:
I would change `app` to `webui` or something that would make it clear what you are running.